packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -244,6 +251,31 @@ ${renderCurrent(outdatedPkg)} ${chalk.grey('=>')} ${renderLatest(outdatedPkg)}`
     .join('\n\n') + '\n'
 }
 
+function renderOutdatedJSON (outdatedPackages: readonly OutdatedPackage[], opts: { long?: boolean }) {
+  const outdatedPackagesJSON = {} as {
+    [outdatedPackageName: string]: {
+      current: string
+      latest: string
+      dependencyType: DependenciesField
+      details?: string
+    }
+  }
+  for (const outdatedPkg of sortOutdatedPackages(outdatedPackages)) {
+    const { packageName, belongsTo } = outdatedPkg
+    outdatedPackagesJSON[packageName] = {
+      current: renderCurrent(outdatedPkg),
```

Comment:
It is a JSON, so I would rather return the unprocessed fields as they are. Like:

```suggestion
      current: outdatedPkg.current,
      wanted: outdatedPkg.wanted,
```

Reply:
Certainly!! This is about JSON format and this is usually used with automation process, so it is better to simplify. [2299772](https://github.com/pnpm/pnpm/pull/5582/commits/2299772b0d1dbac333acbda86ee56ac339eefc8d) :+1: